### ISSUE TITLE
Use accordion component for mobile version of "How it works" section

### ISSUE THREAD

**Issue by ReangeloJ** [oceanprotocol/ocean.js#588]
**Is your feature request related to a problem? Please describe.**
When scrolling on mobile through the "how it works section" (https://oceanprotocol.com/technology/marketplaces), the user experience of the side-tabs becomes annoying. The longer the list, the more a user has to scroll up to see the all text.  Also the text becomes very thin on mobile devices this way

![image](https://user-images.githubusercontent.com/25207023/106895103-ba0fac80-66e7-11eb-89b8-2d22041f96db.png)

**Describe the solution you'd like**
I would suggest using an accordion component with global state where only one accordion can be open at a time. This way when a user scrolls down and decides to open another accordion, the previous one closes, the new one opens and the user can continue their reading in one natural flow. Also this way the text is full width.

**Comment by kremalicious** [comment]
indeed, that looks weird. Not sure an accordion is the best solution but agree this makes content not good to digest. I'll move this into our work queue for our (private) `site` repo
